MSAL/src/native_auth/public/state_machine/error/MSALNativeAuthError.swift

```diff
@@ -26,13 +26,13 @@ import Foundation
 
 @objc
 public class MSALNativeAuthError: NSObject, LocalizedError {
+    public let identifier: String
```

Comment:
Shouldn't there be a doc comment as this is a public member?

Reply:
Yes, done.